src/nrf_to_nrf.h

```diff
@@ -7,6 +7,9 @@
 #ifndef __nrf52840_nrf24l01_H__
 #define __nrf52840_nrf24l01_H__
 #include <Arduino.h>
+#ifndef __MBED__
+#include "Adafruit_TinyUSB.h" //Needed for Serial.print on non-MBED enabled core
+#endif
```

Comment:
```suggestion
#if !defined(__MBED__) || defined(USE_TINYUSB)
// Needed for Serial.print on non-MBED enabled or adafruit-based nRF52 cores
#include "Adafruit_TinyUSB.h" 
#endif
```
You're copy-n-paste skills are too fast for my reading/writing skills.